README.md

```diff
@@ -30,13 +31,28 @@ pip install -r requirments.txt
 - For set up react component set up and running see https://create-react-app.dev/docs/getting-started/
 
 ## Problem and Technical Issues in the course of design
-- Setting up the Heroku app deployment was a huge challenge. Everthing for the deploymet was fine except for linking the Google Calender API credentials to Heroku config var: thid is still a challenge. 
-- Also deleting from the calendar updae was a challenges> it is someworth a difficult task to fetch and delete directly via html from google calendar
+- Setting up the Heroku app deployment was a huge challenge. Everthing for the deploymet was fine except for linking the Google Calender API credentials to Heroku config var: thid is still a challenge but we finally got it done - deployed
```

Comment:
This is not a challenge anymore, we got the google calendar API credentials in Heroku.